ui/imports/shared/popups/ImportCommunityPopup.qml

```diff
@@ -147,14 +147,17 @@ StatusDialog {
                 text: qsTr("Community key")
                 color: Theme.palette.directColor1
             }
-
-            StatusTextArea {
-                id: keyInput
+            StatusScrollView {
```

Comment:
It's working, but I don't understand why..And I'm wondering if we're not missing the root cause in the `StatusTextArea`.

Is it because of `anchors.fill: parent` instead of Layout sizing?

I was expecting the `StatusTextArea` to be self contained and the text area to keep think bounded to the designated size.

Reply:
It's how it works :)

https://doc.qt.io/qt-5/qml-qtquick-controls2-textarea.html#scrollable-textarea